source/forth3/src/word.rs

```diff
@@ -55,6 +55,11 @@ impl Word {
         }
     }
 
+    #[inline]
+    pub fn into_data(self) -> i32 {
```

Comment:
tbh, i would be equally fine with just calling this `data()`. but, up to you, it is technically an into conversion... :woman_shrugging:

Reply:
`Word::data(i32)` is already the (safe) constructor, though that might be equally poorly named.